docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -0,0 +1,192 @@
+# Migration From v3 to v4
+
+<p class="description">Yeah, v4 has been released!</p>
+
+Looking for the v3 docs? [Find them here](https://material-ui.com/versions/).
+
+> This document is a work in progress.
+Have you upgraded your site and run into something that's not covered here?
+[Add your changes on GitHub](https://github.com/mui-org/material-ui/blob/next/docs/src/pages/guides/migration-v3/migration-v3.md)
+
+## Introduction
+
+This is a reference for upgrading your site from Material-UI v3 to v4.
+While there's a lot covered here, you probably won't need to do everything for your site.
+We'll do our best to keep things easy to follow, and as sequential as possible so you can quickly get rocking on v4!
+
+## Why you should migrate
+
+This documentation page covers the *how* of migrating from v3 to v4.
+The *why* is covered in the release blog post: [*Work in progress, on Medium*](https://medium.com/material-ui).
+
+## Updating Your Dependencies
+
+The very first thing you will need to do is update your dependencies.
+
+### Update Material-UI version
+
+You need to update your `package.json` to use the latest version of Material-UI.
+
+```json
+"dependencies": {
+  "@material-ui/core": "^4.0.0-alpha.0"
```

Comment:
This will be outdated with every alpha release. `yarn add @material-ui/core@next` will be up to date at least till release. Or `yarn add @material-ui/core@^4.0.0-alpha` should work to: The caret should match any alpha version.